config/owasp/suppressions.xml

```diff
@@ -52,6 +51,7 @@
     <cve>CVE-2022-45143</cve>
     <cve>CVE-2022-42003</cve>
     <cve>CVE-2022-42004</cve>
+    <cve>CVE-2022-1471</cve>
```

Comment:
we dont need this. can we remove this please.

Reply:
Sure, build was failing bcoz this cve not added in suppression so I have added it.
I will remove it, thanks